templates/sonarqube-values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  tag: 7.6-community
```

Comment:
Is there a restriction to not going to a more recent version?  If there is document here why if links to explanation.